src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const { id, email } = user;
+  const token = jwt.sign({ id, email }, process.env.JWT_KEY);
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
```

Comment:
Returning `null` on verification failure is a good practice, but consider logging the error `e` for debugging purposes. This will help in diagnosing issues with token verification.